api/src/main/resources/firecloud.yaml

```diff
@@ -1011,6 +1011,45 @@ paths:
           description: Internal server error
           content: {}
       x-codegen-request-body-name: profile
+  /register/user/v1/termsofservice:
```

Comment:
mostly copied from firecloud-orchestration: https://cs.github.com/broadinstitute/firecloud-orchestration/blob/9268789279e8c847c84f7c6255e001da9194b4b2/src/main/resources/swagger/api-docs.yaml#L7186

Reply:
Question: yaml file tags will create api controller with the same name, so should the tag name here be words with spaces or with _ or it doesnt matter.

Reply:
I was wondering that too.  But it turns out to work just fine and it generates a `TermsOfServiceApi`